Style clean-up

Ldsmiths opened the pull request:
issue #352

- removed file icons from bottom of GIO page

- cleaned up old Georodeo and GIS Forum pages (fixed minor layout issues, removed broken links or references to missing content)

- Events page - highlighted event (right column) style changed to match GIO page

- removed max-width from navbar scss. This was causing 

- Teacher resources page - edit to tours alert to eliminate overlap with picture

- filterCourses class added to set max-width for training courses dropdown filter (solves issue of arrow not showing)

jwhaney committed:
Merge branch 'develop'

Commit:
style clean-up

issue #352

jwhaney commented on content/markdown/texas-gis-forum/2018/index.md:
```diff
@@ -13,7 +13,6 @@ mainimagesm: https://cdn.tnris.org/images/largest_tree_graphic_sm.jpg
   <a class="btn btn-lg btn-tnris" href="/texas-gis-forum/2018/agenda">View the Agenda</a></p>
 
-<p class="lead">Revisit the forum with our photo recap:<br>
   <h3>Platinum Sponsors</h3>
```
where's the closing `</p>` tag? the html in the markdown is very difficult to read so we just need to be careful here. one missing tag could potentially cause the prod build to fail.

Ldsmiths replied:
I'm guessing there used to be a button or link to view a photo recap here, and maybe when that was removed the closing `</p>` tag got deleted with it. I initially deleted this part since I didn't see any link to the resource it was referring to. 

I looked back through this file and there was some wonky stuff going on with `<p>` tags all over it... closing tags before any opening tags, open tags that never got closed, tags used in places with no text. My latest push should solve all of these.

jwhaney committed:
tab/spacing

jwhaney committed:
spacing

jwhaney commented on static/js/gio_calendar_next_cm.js:
```diff
@@ -19,23 +19,33 @@ function retrieveNextCommunityMeeting() {
+        <br>
+      </h3>
+      <h5>
+        ${e.pretty_date}<i class="fa fa-star" style="color: #1e8dc1;"></i>
```
this font awesome star icon doesn't seem to be showing up that i can see

Ldsmiths replied:
<img width="1430" alt="Screen Shot 2021-04-28 at 1 47 11 PM" src="https://user-images.githubusercontent.com/70979203/116460053-34a22100-a82c-11eb-8cd4-201ac099027f.png">

We can't see this section now since it only shows events checked as community meetings and the Q2 meeting was just yesterday. I tested in the screenshot above by taking the community meeting requirement out of the API url and it seems to be showing

jwhaney commented on static/js/gio_calendar_next_cm.js:
```diff
@@ -19,23 +19,33 @@ function retrieveNextCommunityMeeting() {
       // check if various properties exist, and set variables if so
       var timeRange = e.pretty_time ? `<time><i class="fa fa-time"></i> ${e.pretty_time}</time><br>` : '';
```
is this time fa icon showing either? hmm

Ldsmiths replied:
This fa icon needed to be updated to show (see screenshot above) but I don't know that it's necessary. It's not being used on the GIO page. I removed it, but we can add it back if we think it's valuable.

jwhaney replied:
![Screen Shot 2021-04-29 at 8 59 59 AM](https://user-images.githubusercontent.com/13525011/116563290-83ea5f00-a8c9-11eb-8473-0b152f34e7a7.png)

it's in the code for the gio page, but not showing

jwhaney replied:
i would vote just to remove it

Ldsmiths replied:
It could be updated to `<i class="far fa-clock"></i>` but I was thinking of removing it there too, unless we think it's necessary.

jwhaney committed:
remove br tags, spacing/tabs

jwhaney reviewed (changes requested):
couple small things to check on. see comments.

Commit:
minor edits

-added closing <p> tags to 2018 forum page
-changes to icons and formatting of community meeting event box

Commit:
removed fa icon from gio page event boxes

Commit:
more clean up of old gis forum pages

- Fixed spacing for md formatting

- Removed double <hr>’s

- Edited layout to fix 2015 banner not showing

jwhaney reviewed (approved):
👍🏻
